src/App/App.js

```diff
@@ -35,10 +38,24 @@ class App extends Component {
     }
```

Comment:
Generally better to keep your lifecycle methods not bound. You can change this function by putting the async before the componentDidMount, like this:
async componentDidMount() {
   ...
}

Great async/await with try/catch usage!!